src/waveform/renderers/glwaveformrendererrgb.cpp

```diff
@@ -61,7 +64,7 @@ void GLWaveformRendererRGB::draw(QPainter* painter, QPaintEvent* /*event*/) {
     float allGain(1.0), lowGain(1.0), midGain(1.0), highGain(1.0);
     getGains(&allGain, &lowGain, &midGain, &highGain);
 
-    const float kHeightScaleFactor = 255.0 / sqrtf(255 * 255 * 3);
+    const float kHeightScaleFactor = static_cast<float>(255.0 / sqrtf(255 * 255 * 3));
```

Comment:
this should become a static or a global constant. 
Th cast can be omitted with 255.0f